ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/PrivacySendTransaction.java

```diff
@@ -113,7 +108,7 @@ public JsonRpcResponse getResponse() {
     }
   }
 
-  protected interface AfterTransactionValid {
+  public interface AfterTransactionValid {
```

Comment:
nit: could have a @FunctionalInterface
nit: Probably badly named, the class is just ResponseSupplier - which get USED if a Transaction is valid....